src/lambdacd_cctray/core.clj

```diff
@@ -43,15 +43,22 @@
         formatted (f/unparse (f/formatters :date-time) most-recent-update)]
     formatted))
 
+(defn- get-add-prefix-with-default [config]
```

Comment:
I think this could be simplified by using `get` with a default parameter, e.g. 
```clojure
(get config :cctray-add-prefix true)
````
Probably doesn't need to be its own function any more then.